main/src/ca/uwaterloo/flix/language/ast/KindedAst.scala

```diff
@@ -166,7 +166,7 @@ object KindedAst {
 
     case class NewObject(name: String, clazz: java.lang.Class[_], methods: List[KindedAst.JvmMethod], loc: SourceLocation) extends KindedAst.Expression
 
-    case class NewChannel(exp: KindedAst.Expression, elmType: Type.Var, loc: SourceLocation) extends KindedAst.Expression
+    case class NewChannel(reg: KindedAst.Expression, exp: KindedAst.Expression, elmType: Type.Var, loc: SourceLocation) extends KindedAst.Expression
```

Comment:
While this may seem counter-intuitive, we usually just call our exps `exp1` and `exp2` just using the position to know which is which.

This is actually somewhat sensible since expressions are almost always treated uniformly.

Reply:
Ah, understood. I'll revise.